Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticTypes.cs

```diff
@@ -9,24 +9,21 @@
 
 namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
 {
-
     /// <summary>
-    /// Properties of the extent in which a damped
-    /// harmonic oscillator is free to move.
+    /// Properties of a linear, one-dimensional extent
+    /// in which a damped harmonic oscillator is free to move.
     /// </summary>
     [Serializable]
-    public struct ElasticExtentProperties<T>
+    public struct LinearElasticExtent
     {
         /// <value>
-        /// Represents the lower bound of the extent,
-        /// specified as the norm of the n-dimensional extent
+        /// Represents the lower bound of the extent.
```

Comment:
Since this is now specific to a single dimension, I'm curious if there's any way to specify if there's any guidance/limits to the ranges (i.e. do they generally represent a percent of the thing being modified, or meter units, etc). Or if it's really specific to each application, that would also be helpful to understand.

Reply:
The elastic dimensions have no physical meaning/relationship until the elastic system is actually hooked up to something.

A "linear" elastic system can be used to drive euler rotations, too, or the brightness of a light, or the color of a material.